### ISSUE TITLE
interactive example on mybinder not working

### ISSUE THREAD

**Issue by PeerHerholz** [nipy/niwidgets#55]
Ahoi hoi there,

Problem:

I was just trying to run the examples on mybinder, but appears to be a problem as none of the `imports` do work:
![Screen Shot 2019-10-16 at 9 30 34 AM](https://user-images.githubusercontent.com/20129524/66924474-b3dc5500-eff8-11e9-87c2-3ea62471d068.png)

Steps to reproduce:

Try to run the examples on mybinder [here](https://mybinder.org/v2/gh/nipy/niwidgets/master?filepath=index.ipynb).

Possible solution:

No precise idea, but seems that the examples are running on a local mybinder instance. Hence, maybe check if everything is okay there.

Happy to help solving this in case there's something I could do.

Best, Peer

**Comment by satra** [comment]
it looks like the package installer was reorganized in a previous commit. and as such the automated binder builder is not working. perhaps @janfreyberg  can provide some information. 

otherwise one needs to create a `.binder` folder  to add some binder specific action.

**Comment by janfreyberg** [comment]
Yeah, this is a really good catch. I updated to use a different install system (poetry) and that now causes problems on binder. I don't have a huge amount of experience with binder - I'll look into it but am not sure how long it will take me!

**Comment by PeerHerholz** [comment]
I'm not sure if I can help, but I would be happy to give it a try. @satra would a Dockerfile with the respective settings be an overkill? That's how I usually run tutorials, etc. on binder.  However, I also see the feasibility of a `.binder`.

**Comment by satra** [comment]
@PeerHerholz - sorry for the really late reply. but given that many projects are using poetry, this may be a question for the binder team whether they plan to support poetry, before thinking of other solutions.
